docs/core/fallback-actions.rst

```diff
@@ -99,30 +86,74 @@ the same manner as the ``FallbackPolicy``).
+    - If the user denies by sending a message with the intent ``out_of_scope``, the
+      original intent is classified as the specifies ``deny_suggestion_intent_name``,
+      and an ultimate fallback action ``fallback_nlu_action_name`` is
+      triggered (e.g. a handoff to a human).
 
-Rasa Core provides the default implementations of
+Rasa Open Source provides default implementations for
 ``action_default_ask_affirmation`` and ``action_default_ask_rephrase``.
 The default implementation of ``action_default_ask_rephrase`` action utters
-the response ``utter_ask_rephrase``, so be sure to specify this
+the response ``utter_ask_rephrase``, so please make sure to specify this
 response in your domain file.
-The implementation of both actions can be overwritten with :ref:`custom actions <custom-actions>`.
+The implementation of both actions can be overwritten with
+:ref:`custom actions <custom-actions>`.
 
-You can specify the core fallback action as well as the ultimate NLU
-fallback action as parameters to ``TwoStageFallbackPolicy`` in your
-policy configuration file.
+To use the ``Two-Stage-Fallback`` for messages with low NLU confidence, add the
+following :ref:`rule<rules>` to your training data. This rule will make sure that the
+``Two-Stage-Fallback`` will be activated whenever the user send a message which receives
+low classification confidence.
 
 .. code-block:: yaml
 
-    policies:
-      - name: TwoStageFallbackPolicy
-        nlu_threshold: 0.3
-        core_threshold: 0.3
-        ambiguity_threshold: 0.1
-        fallback_core_action_name: "action_default_fallback"
-        fallback_nlu_action_name: "action_default_fallback"
-        deny_suggestion_intent_name: "out_of_scope"
+    rules:
+
+    - rule: Implementation of the Two-Stage-Fallback
+      steps:
+      - ...
+      - intent: nlu_fallback
+      - action: two_stage_fallback
+      - form: two_stage_fallback
+
+Handling Low Core Confidence
+----------------------------
+
+Similar as users might send unexpected messages,
+it is also possible that their behavior might lead them down unknown conversation paths.
+Rasa's machine learning policies such as the :ref:`ted_policy` are optimized to handle
+these unknown paths.
+
+To handle cases where even the machine learning policies can't predict the
+next action with confidence, make sure to add the :ref:`rule-policy` to your
+policy configuration. The :ref:`rule-policy` will predict a default action if no
+:ref:`policy<policies>` has a prediction for the next
+action of the assistant which crosses a configurable threshold.
+
+You can configure the action which is run in case low of Core confidence as well as
+the threshold for this as follows:
+
+.. code-block:: yaml
+
+  policies:
+    - name: RulePolicy
+      # Confidence threshold for the `fallback_action_name` to apply.
+      # The action will apply if no other action was predicted with
+      # a confidence >= core_fallback_threshold
+      core_fallback_threshold: 0.4
```

Comment:
this is not currently implemented right?

Reply:
No. I stated that in the PR description.